docs/examples/CMS40B-E--form.js

```diff
@@ -0,0 +1,129 @@
+  subTitle: 'CMS40B-E',
+  formId: '',
+  urlPrefix: '/',
+  submitUrl: '',
+  introduction: Introduction,
+  confirmation: '',
+  defaultDefinitions: {},
+  preSubmitInfo: {
+    notice: 'I hope you did not lie!',
+    required: true,
+    label: 'I swear this information is correct to the best of my knowledge.',
+    field: 'signed',
+    error: 'You must check the box to submit this form and apply for Medicare Part B'
+  },
+  chapters: {
+    CP1: {
+      pages: {
+        PG1: {
+          path: 'verification',
+          title: 'Enrollment verification',
+          uiSchema: {
+            signUp: {
+              'ui:title': 'I wish to apply for Medicare Part B',
+              'ui:options': {
+                expandUnderClassNames: 'schemaform-expandUnder-indent'
+              }
+            },
+            'view:preQualification': {
+              'ui:description': 'To apply for Medicare Part B, you must have already applied for and recieved Medicare Part A. If you do not have Part A and want to sign up, please contact Social Security at 1-800-772-1213. TTY users should call 1-800-325-0778.',
+              'ui:options': {
+                expandUnder: 'signUp'
+              }
+            },
+            medicareNumber: {
+              'ui:title': 'Medicare Number',
+              'ui:description': 'Your Part A paperwork has a Medicare number in the form XXXX-XXX-XXXX consisting only of letters and digits, please enter it below.',
+              'ui:errorMessages': {
+                pattern: 'Your Medicare number should be in the form XXXX-XXX-XXXX, letters and digits separated by dashes'
+              }
+            }
+          },
+          schema: {
+            type: 'object',
+            required: [ 'signUp', 'medicareNumber' ],
+            properties: {
+              signUp: {
+                type: 'boolean'
+              },
+              'view:preQualification': {
+                type: 'object',
+                properties: {}
+              },
+              medicareNumber: {
+                type: 'string',
+                pattern: '[0-9a-zA-z]{4}-[0-9a-zA-z]{3}-[0-9a-zA-z]{4}'
+              }
+            }
+          }
+        },
+        PG2: {
+          path: 'enrollee-info',
+          title: 'Enrollee information',
+          uiSchema: {
+            zip: {
+              'ui:errorMessages': {
+                pattern: 'Please enter a 5-digit zip code or 9-digit zip+4 code'
+              }
+            },
+            phone: {
+              'ui:errorMessages': {
+                minLength: 'Please enter a phone number, including area code'
+              }
+            }
+          },
+          schema: {
+            type: 'object',
+            required: [ 'name', 'address', 'city', 'state', 'zip', 'phone' ],
+            properties: {
+              name: {
+                type: 'string',
+                title: 'Full name (Last, First, Middle)',
```

Comment:
This is very interesting, usually I use `ui:title` under `uiSchema`, which you used above. I don't think it matters, seems like they're both treated the same, but I wonder if we should consistently use one over the other?

Reply:
It works with either. In places where it saved having an entry at all in `uiSchema` for that field I put it into `schema` to shorten things up. We can use `ui:title` consistently, although it does point out the verbosity of the current setup.

Reply:
Looking at some of these forms I see I could have saved even more than I did by going all-`title` in more places, like [here](https://github.com/usds/us-forms-system/pull/271/files#diff-311ffb3b6f74e31e3c3b94c1d2387eedR19). It really shows how much noisier it is to have two sections instead of one.

Reply:
Yes, good point. In that case totally fine to leave `title` for now and then we can revisit this when we reimagine the config :)